config/projects.yml

```diff
@@ -5,7 +5,6 @@ admin-bundle:
       versions:
         symfony: ['3.4']
         sonata_core: ['3']
-        sonata_block: ['3']
```

Comment:
Okay now we can discuss on a diff. :stuck_out_tongue: 

Can't we just use `4`? Normally it should work with branch alias.

The thing that cause issue to me is to just remove sonata_block. We should test at least one version IMO.

Reply:
A new version of AdminBundle is incompatible with current stable BlockBundle and can work only with a new version of BlockBundle, so i think it is not an option. I removed dependency on `^3.11` from composer.json and left only `^4.0@dev` for `sonata-project/block-bundle` dependency in my PR for AdminBundle, see diff https://github.com/sonata-project/SonataAdminBundle/pull/4938/files.

DashboardBundle, DoctrinePhpcrAdminBundle, FormatterBundle, PageBundle, SeoBundle and TimelineBundle has direct dependency on BlockBundle too. When i will update these bundles, i will update that dependency too

Reply:
@covex-nn Indeed, if the deps is already here, it will change nothing for now. But it would be better to keep it with only `4` in order to remember to test multiple versions in the future IMHO.

What do you think?

Reply:
@Soullivaneuh a line with `sonata_block: ['4']` will generate two jobs for builds, see [.travis-ci.yml](https://github.com/sonata-project/SonataAdminBundle/blob/master/.travis.yml):

1. with `env: SONATA_BLOCK=4.*`
2. with `env: SONATA_BLOCK='dev-master as 4.x-dev'` (this job is allowed to fail)

These jobs first try to require another SonataBlockBundle (they take a version from SONATA_BLOCK env var) and then do all other stuff. See [before_install_test.sh.twig](https://github.com/sonata-project/dev-kit/blob/master/project/.travis/before_install_test.sh.twig)

BlockBundle v4 was not relesed yet. So, after updating `sonata_block: ['3']` to `sonata_block: ['4']` all builds will always fail because of job with `SONATA_BLOCK=4.*`.

I think, that such jobs should be added only for packages, that really has multiple versions. But there won't be multiple versions for SonataBlockBundle, there will be just `4`, because bundles will not be not compatible with lower versions =) Also, i think that all jobs `SONATA_BLOCK` and `SONATA_CORE` are useless now, because everything was already tested with regular jobs

Reply:
@Soullivaneuh is there anything else that I could do here?

Reply:
@covex-nn Sorry for the time but I try to understand the need: Any project should work with at least one stable dependency. It's not the case here: https://github.com/sonata-project/SonataAdminBundle/pull/4938/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R29

If we have to release a new major of SonataAdminBundle, what should we do? At least, it should works with both versions or block bundle v4 should be release first.

Also, to test non-released dependencies, we already have the dev-master alias. But we may update the travis twig file to require 3.*@dev each time.